### ISSUE TITLE
minify builds of jupyterlab

### ISSUE THREAD

**Issue by lukemarsden** [dotmesh-io/jupyterlab-tensorflow#40]
https://github.com/dotmesh-io/jupyterlab-tensorflow/pull/39 attempted to minify jupyterlab, but failed...

```
luke@boom:~/ps/local-stack$ wget https://d10769ac.app.cloud.dotscience.com/static/lab/vendors~main.663127da24f92e98775c.js
--2019-11-05 20:33:25--  https://d10769ac.app.cloud.dotscience.com/static/lab/vendors~main.663127da24f92e98775c.js
Resolving d10769ac.app.cloud.dotscience.com (d10769ac.app.cloud.dotscience.com)... 64:ff9b::23cd:efe1, 35.205.239.225
Connecting to d10769ac.app.cloud.dotscience.com (d10769ac.app.cloud.dotscience.com)|64:ff9b::23cd:efe1|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 29039885 (28M) [application/javascript]
```

**Comment by lukemarsden** [comment]
https://github.com/dotmesh-io/jupyterlab-tensorflow/pull/41 seems to fare better, testing in prod now

**Comment by itamarst** [comment]
Fixed by #41.
